helpers/python.ts

```diff
@@ -51,6 +52,9 @@ const getAdditionalDependencies = (
       dependencies.push({
         name: "llama-index-vector-stores-pinecone",
         version: "^0.2.1",
+        constraints: {
+          python: ">=3.11,<3.12",
+        },
```

Comment:
We can specify a specific Python version as a dependency constraint; see: https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies